lib/content/pkg.json

```diff
@@ -2,7 +2,7 @@
   "author": "GitHub Inc.",
   "files": {{{ json distPaths }}},
   "scripts": {
-    "lint": "eslint \"**/*.js\"",
+    "lint": "{{#if eslint}}eslint \"**/*.js\"{{else}}exit 0{{/if}}",
```

Comment:
```suggestion
    "lint": "{{#if eslint}}eslint \"**/*.js\"{{else}}echo linting disabled{{/if}}",
```
wdyt?

Reply:
lgtm

Reply:
Committed and squashed into the eslint commit.